packages/nextjs/components/scaffold-eth/Address.tsx

```diff
@@ -76,6 +77,10 @@ export const Address = ({ address, disableAddressLink, format }: TAddressProps)
       </div>
       {disableAddressLink ? (
         <span className="ml-1.5 text-lg font-normal">{displayAddress}</span>
+      ) : blockExplorerAddressLink.includes("blockexplorer") ? (
```

Comment:
I think we could have a less hacky way of doing this. We are already using `getTargetNetwork()` in this file, so we could rely on it (check if we are in the local network).